lib/screens/auth/signup_screen.dart

```diff
@@ -429,6 +429,7 @@ class _SignupScreenState extends State<SignupScreen> {
                                                           DesignColors.black),
                                             ),
                                             Switch(
+                                              key: const Key("agbSwitch"),
```

Comment:
`agbSwitch` is a funny denglisch word ;) something like `termsAndConditionsSwitch` is lengthy, but less confusing